Untitled-1.html

```diff
@@ -0,0 +1,39 @@
+<DOCTYPE html>
+	<html lang="en">
```

Comment:
Tutaj chyba niepotrzebny odstęp?

Ogólna zasada dla wszystkich wcięć w plik. każdy nowy "poziom" to wcięcie za pomocą klawisza TAB

Reply:
<html jest na tym samym poziomie co <DOCTYPE html> więc powinno być na równo